devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy

```diff
@@ -441,20 +455,18 @@ def humidityEvent(humidity) {
 	def hum = device.currentState("humidity")?.value
 	if(!hum.equals(humidity)) {
         log.debug("UPDATED | Humidity is (${humidity}) | Original State: (${hum})")
-		sendEvent(name:'humidity', value: humidity, unit: "%", descriptionText: "Humidity is ${humidity}" , displayed: true, isStateChange: true)
+		sendEvent(name:'humidity', value: humidity, unit: "%", descriptionText: "Humidity is ${humidity}" , displayed: false, isStateChange: true)
     } else { Logger("Humidity is (${humidity}) | Original State: (${hum})") }
 }
 
 def presenceEvent(presence) {
-	def val = getPresence()
```

Comment:
No issue - just my not wanting to get confused between presence and nestPresence


On Mar 23, 2016, at 9:32 AM, Tonesto7 <notifications@github.com<mailto:notifications@github.com>> wrote:


In devicetypes/tonesto7/nest-thermostat.src/nest-thermostat.groovy<https://github.com/tonesto7/nest-manager/pull/34#discussion_r57158762>:

>      } else { Logger("Humidity is (${humidity}) | Original State: (${hum})") }
>  }
>
>  def presenceEvent(presence) {
> -     def val = getPresence()


Purely out of curiosity was their an issue with calling the getPresence() vs. the device.currentState?

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/tonesto7/nest-manager/pull/34/files/71738294e6f15465e5de7ddacf369c0b6f73c063#r57158762>